### ISSUE TITLE
[Windows] Button text and button semantic text get focused separately

### ISSUE THREAD

**Issue by rachelkang** [dotnet/maui#5107]
### Description

On Windows, the button text gets focused separately from the button semantic text (i.e. SemanticHint), preventing them from getting read aloud together and creating a disjointed experience. The button text should be surfaced up to the same focus level as the semantic text and they should all be read aloud together.

### Steps to Reproduce

1. Launch [TestPR4524.zip](https://github.com/dotnet/maui/files/8200955/TestPR4524.zip) on Windows
2. Turn on Narrator
3. Tab focus will go to the Button and read the semantic text, but the button text "Click me" itself won't get read aloud. "Click me" only gets read aloud, and on its own, when touch swiping (one finger swipe right) into the button
 

### Version with bug

Preview 13 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

WinUI

**Comment by rachelkang** [comment]
This is also happening for various other controls on Windows (i.e. FlyoutMenuItem)
